frontend/src/pages/RegionalDashboard/index.js

```diff
@@ -39,113 +41,97 @@ export default function RegionalDashboard({ user }) {
   });
 
   const regions = getUserRegions(user);
+  const defaultRegion = hasCentralOffice ? 14 : regions[0];
 
-  // eslint-disable-next-line max-len
-  const [appliedRegion, updateAppliedRegion] = useState(hasCentralOffice ? 14 : regions[0]);
-  const [dateRange, updateDateRange] = useState(defaultDate);
-
-  // this can be killed when we add the new filters to this page
-  const [roleFilter, updateRoleFilter] = useState();
-
-  /*
-    *    the idea is that this filters variable, which roughly matches
-    *    the implementation on the landing page,
-    *    would be passed down into each visualization
-    */
-
-  const filters = [
+  const [filters, setFilters] = useUrlFilters([
     {
       id: uuidv4(),
       topic: 'region',
       condition: 'Contains',
-      query: appliedRegion,
+      query: defaultRegion ? defaultRegion.toString() : '',
     },
     {
       id: uuidv4(),
       topic: 'startDate',
       condition: 'Is within',
-      query: dateRange,
+      query: defaultDate,
     },
-  ];
+  ]);
+
+  const startDateFilter = filters.find((filter) => filter.topic === 'startDate');
```

Comment:
Do we still need to show the date range on filters in the regional TTA page? Or do we need to start on the other pages? We don't do it on any other page. Seems kinda of strange to show only a single filter condition stated on the widget when we add the ability to filter on so may other things.

Reply:
Good point, I'll ask about this at the UI meeting